# Define how to read serial
nmea_serial_driver_node:
  port: /dev/ttyUSB0
  baud: 115200

nmea_parser_node:
  GPFPD: true
  GTIMU: false
  GPGGA: false
  NVSTD: false
  GPCHC: true

# Used for NavsatFix & Imu message if "use_gnss_time" is true.
time_set:
  time_zone: 8  # parse gnss time to ROS NavsatFix stamp, if use GPGGA defalut time zone is 0
  leap_second: 47 #  used for GPFPD: leap second from 1970s to 2022 is 27

NavsatFix_generator_node:
  NavsatFix_generate_from: GPCHC  # optional: GPCHC, GPFPD, GPGGA
  use_gnss_time: true


Imu_generator_node:
  Imu_generate_from: GPCHC  # optional: GPCHC, GPFPD-GTIMU
  use_gnss_time: true

# In case of gnss point refer to imu original point. No external matrix.
Odometry_generator_node:
  coordinate_type: ENU  # optional: ENU, LLA
  frame_id: map